### ISSUE TITLE
 torchvision.datasets.cifar has no training labels, test labels

### ISSUE THREAD

**Issue by pavanteja295** [GT-RIPL/Continual-Learning-Benchmark#2]
Hi,
I'm using pytorch=1.1.0 and torchvision=0.3.0. For this current version there is a small bug in the cifar100 and cifar10 data loaders.

It seems to be that cifar class has no attribute training_labels or test_labels which was used in dataloaders/base.py. Also the documentation states no such attributes [CIFAR](https://pytorch.org/docs/stable/_modules/torchvision/datasets/cifar.html#CIFAR10).  However MNIST has such attributes.

Is this is a bug or was it working for you for the previous versions? It's strange that torchvision lacks homogenity among  datasets.

Lemme know if it's a bug I can raise a pull request

**Comment by yenchanghsu** [comment]
The original implementation requires torchvision=0.2.2.
Please pull this commit to support torchvision 0.3.0 : 9fca67d18fa9ca2dee37fabffeb1e4435af83d9e (It is backward-compatible)
I also added the scripts for cifar100 for your convenience: 2e01902c97cc7cff1f9e303f9874221ff6273c8a

**Comment by pavanteja295** [comment]
Thanks a lot I shall use it !
